arthur/worker.py

```diff
@@ -35,10 +35,22 @@
 class ArthurWorker(rq.Worker):
     """Worker class for Arthur"""
 
+    def __init__(self, queues, **kwargs):
+        super().__init__(queues, **kwargs)
+        self.__pubsub_channel = CH_PUBSUB
+
+    @property
+    def pubsub_channel(self):
+        return self.__pubsub_channel
+
+    @pubsub_channel.setter
+    def pubsub_channel(self, value):
+        self.__pubsub_channel = value
+
     def prepare_job_execution(self, job):
         # Fixes the error #479 of RQ. Remove it as soon as it gets fixed.
         # (https://github.com/nvie/rq/issues/479)
-        rq.push_connection(self.connection)
+        # rq.push_connection(self.connection)
```

Comment:
This issue seems indicated to be fixed upstream.  Have not observed any operational issues so far.

Reply:
I think this can be safely removed. I haven't seen that error in a while.

Reply:
Removed.